### DISCUSSION TITLE
Problem with sending logs to Cloudwatch log groups. How to properly config the monolog to send logs to Cloudwatch log group (using path: 'php://stdout')

### DISCUSSION THREAD

**Discussion by westacton-ronald** [symfony/symfony#51262] [Q&A]
Hi, 

I want to inquire about sending logs to the Cloudwatch log group via `path: 'php://stdout'` using monolog.

My Setup is the following:
* Symfony 5.4
* Running Symfony-based system from AWS ECS (container)
* with AWSlogs driver task def setup in ECS (apache logs is successful with dumping logs to stdout)

my current and running monolog.yaml is setup with the following:

       main:
            type: stream
            path: php://stdout
            level: info
            channels: ["!event"]
            formatter: monolog.formatter.json
            
        requests:
            type: stream
            path: php://stdout
            level: debug
            channels: ["requests"]
            formatter: monolog.formatter.json`

Problem:
* NO Symfony logs are sent to the Cloudwatch log groups.
Tried also changing the path to php://stderr but still the same no logs are received.

Troubleshoot:
* created a simple PHP script with the following to test if php can do a stdout. Yes, can receive this message from Cloudwatch.
meaning the PHP config is working fine
`$stdout = fopen("php://stdout", "w");`
`fwrite($stdout,"Simple PHP Script to STDOUT:\n");`

* from Docker container this monolog.yaml config works fine. as it displays the logs to the docker console log without any problem.

* also edit the php.ini with `log_errors = On` and `error_log = /dev/stdout` still not working with logging to Cloudwatch

Can someone, help me with this implementation? Thanks in advance

**Comment by NelsonLopez69** [comment]
@westacton-ronald  could you solve it?  I am dealing with the same issue
